src/components/accordion/accordion.test.tsx

```diff
@@ -224,7 +224,9 @@ describe('OuiAccordion', () => {
       // click button
       component.find('button').simulate('click');
 
-      expect(childWrapper).toBe(document.activeElement);
+      setTimeout(() => {
+        expect(childWrapper).toBe(document.activeElement);
+      }, 0);
```

Comment:
How does the `setTimeout` fix the issues? I remember seeing changes about spying on the `focus` event, why don't we do that? It seems less flaky to me

Reply:
I don't know the exact reason this changed between jsdom versions, but previously you could check for the activeElement immediately, whereas with the update you need to await an update cycle for focus changes to be applied. (There are probably other possible ways of doing this that may be better, such as `jest.useFakeTimers();` and `jest.advanceTimersByTime(0);`). But I found that the setTimeout method worked consistently in all cases, including some which were previously skipped, with minimal changes to existing test structure.

Before I got this working, I was spying on the focus event, but that's actually worse, because it only tells you that the event was triggered, but mostly we want to test where the focus actually ends up, which is better tested via `document.activeElement`.

Reply:
How confident are we that we are not introducing flakiness?

Have you by chance experimented with https://github.com/enzymejs/enzyme/issues/2337#issuecomment-615131784?